kernel/dev/rtc/rtc.cpp

```diff
@@ -136,23 +142,52 @@ void read_rtc() {
     }
 
     // Calculate the full (4-digit) year
-    if(RTC_CURRENT_CENTURY != 0) {
+    if (RTC_CURRENT_CENTURY != 0) {
         rtc_year += rtc_century * 100;
     } else {
         rtc_year += (RTC_CURRENT_YEAR / 100) * 100;
-        if(rtc_year < RTC_CURRENT_YEAR) rtc_year += 100;
+        if (rtc_year < RTC_CURRENT_YEAR)
+            rtc_year += 100;
     }
 }
 
-void rtc_print() {
+/*
+* JDN is time since noon Universal Time on January 1, 4713 BCE (on the Julian calendar)
+* Developed by Fliegel and van Flandern (1968), published in "Compact Computer Algorithms"
+* 
+*           (153 * m) + 2                y     y     y
+* JDN = d + ------------- + (365 * y) + --- - --- + --- - 32045
+*                  5                     4    100   400
+*           <  Part #1  >  < Part #2 >  <   Part #3   > < Part #4 >
+
```

Comment:
Missing a `*` here and at the beginning (`/**` instead of `/*`). Otherwise looks good!